core/src/types.ts

```diff
@@ -266,3 +266,53 @@ export declare interface DebtAuctionEnriched extends DebtAuctionActive {
 }
 
 export declare interface DebtAuctionTransaction extends DebtAuctionEnriched, CompensationAuctionTransactionFees {}
+
+type CollateralType = CollateralConfig['title'];
+
+export declare interface LiquidationLimits {
+    maximumProtocolDebtDai: BigNumber;
+    maximumCollateralDebtDai: BigNumber;
+    currentProtocolDebtDai: BigNumber;
+    currentCollateralDebtDai: BigNumber;
+}
+
+export declare interface VaultCollateralParameters extends LiquidationLimits {
+    stabilityFeeRate: BigNumber;
+    maxDebtPerCollateralUnit: BigNumber;
+}
+
+export declare interface VaultBase {
+    id: number;
+    address: string;
+    collateralType: CollateralType;
+}
+
+export declare interface VaultAmount {
+    initialDebtDai: BigNumber;
+    collateralAmount: BigNumber;
+}
+
+export declare interface VaultTransactionFees {
+    transactionFeeLiquidationEth: BigNumber;
+    transactionFeeLiquidationDai: BigNumber;
+}
+
+export declare interface VaultBaseInfo extends VaultBase, VaultAmount {}
```

Comment:
Let's call this `Vault` then?
```suggestion
export declare interface Vault extends VaultBase, VaultAmount {}
```
Otherwise [the suggestion](https://github.com/sidestream-tech/unified-auctions-ui/pull/446/files#r963578099) is not fully implemented